src/Stratis.Features.FederatedPeg/Wallet/FederationWalletManager.cs

```diff
@@ -877,16 +877,18 @@ private bool TransactionHasValidUTXOs(Transaction transaction, List<Coin> coins
             }
         }
 
-        /// <inheritdoc />
-        public int CompareOutpoints(OutPoint outPoint1, OutPoint outPoint2)
+        /// <summary>
+        /// Compares two outpoints to see which occurs earlier.
+        /// </summary>
+        /// <param name="outPoint1">The first outpoint to compare.</param>
+        /// <param name="outPoint2">The second outpoint to compare.</param>
+        /// <returns><c>-1</c> if the <paramref name="outPoint1"/> occurs first and <c>1</c> otherwise.</returns>
+        internal int CompareOutpoints(OutPoint outPoint1, OutPoint outPoint2)
```

Comment:
I see that `CompareOutpoints` is only ever called from CCTS so why not just move it there?

Reply:
Its used by `FindWithdrawalTransactions`.